### ISSUE TITLE
Add pretrained models

### ISSUE THREAD

**Issue by breandan** [mozilla/DeepSpeech#435]
Is it possible to add links to some pretrained models? I would like to test the performance on some real world speech, but could not find any reference to these in the docs. Thanks!

**Comment by kdavis-mozilla** [comment]
We're planning on adding links to some pre-trained models as soon as we're satisfied with their word error rate (WER).

We're targeting a WER of at max 10% on the TED test set and hope that we can, within the next few weeks, release a model.

**Comment by gvoysey** [comment]
@kdavis-mozilla is there any update on this line of work?

**Comment by kdavis-mozilla** [comment]
@gvoysey I wish I could give you the models right now. But I can't. Sorry. The WER still needs to be tuned.

We're still waiting on our new hardware which will allow us to tune hyperparameters with a quicker turn around. But it's not here yet.

Unfortunately the ETA for the hardware looks to be about 4 weeks out now. Then, once we have the hardware, we'll still need to tune for about a week or two.

Again, sorry.

**Comment by gvoysey** [comment]
@kdavis-mozilla no worries.  I have been thrashing some GPUs myself (on librivox), hopefully they'll finish soon!

**Comment by jacobjennings** [comment]
I'm curious how large a trained model from a large dataset like TED is on disk, as a rough estimate?

**Comment by gvoysey** [comment]
I have a trained librivox model which is ~800 MB.  I've found deepspeech2 implementations on other systems (torch) that are ~600 MB.    These facts are anecdata. :)

**Comment by phasnox** [comment]
You know how long will it take to train with the TED dataset, if I have a GTX 1080?

**Comment by kdavis-mozilla** [comment]
@phasnox We use 4 Titan X's and it takes about 5 days. So, I'd guess on the order of 20 days.

**Comment by getnamo** [comment]
@kdavis-mozilla maxwell Titan X, Titan X (Pascal), or Titan Xp? I hate that we need the distinction...

**Comment by gvoysey** [comment]
and off by one errors 😇

**Comment by striki70** [comment]
Any news for links to trained models?

**Comment by striki70** [comment]
@phasnox 20 days for how many epochs? WER achieved?

**Comment by phasnox** [comment]
@striki70 yehp WER achieved would be nice. But I have'nt try yet, I need to get a better cooling system.

**Comment by gvoysey** [comment]
@kdavis-mozilla just checking in on any updates you have on trained models.

**Comment by kdavis-mozilla** [comment]
Just got the hardware and installed it last week. Now we're working through a few OOM issues which appear in the cluster setting and not in the single node setting. So, it's a work in progress.

**Comment by pythonmobile** [comment]
Waiting for it :) Thanks.

**Comment by AnimeshKoratana** [comment]
Are there any updates regarding the pretrained models?

**Comment by reuben** [comment]
We're training a bunch of models and getting ready to share them. It's coming soon! :)

**Comment by ThejanW** [comment]
Are they ready now?

**Comment by reuben** [comment]
We'll comment here when they're available.

**Comment by gardenia22** [comment]
I test TED dataset with Google API using this tool [autosub](https://github.com/agermanidis/autosub) with little modification. Google API achieved 27.3162% WER. Is 10% WER aiming too high?

**Comment by reuben** [comment]
That's off topic for this issue. Please join our IRC channel for questions and discussions. I'm locking this issue to try and get people to stick to the proper communication channels.

**Comment by reuben** [comment]
We have a first release of an American English model available in our releases page: https://github.com/mozilla/DeepSpeech/releases/latest

Please check it out and experiment, we're excited to see what you can do with it. We also set up discussion forums on Discourse, check the release notes for links.

**Comment by saikishor** [comment]
@reuben We surely appreciate the work and effort you have taken in, to provide us the model. It works amazing. It would be much more helpful, if you could provide us the checkpoints of the above model.

**Comment by nicolaspanel** [comment]
@reuben @kdavis-mozilla and others, thanks for the great work (amazing, really !)
Do you plan to share pre-trained models for other languages as well in future releases (ex: french) ?

**Comment by kdavis-mozilla** [comment]
@nicolaspanel Thanks, and yes we want to share models for as many languages as we can!

**Comment by nicolaspanel** [comment]
@kdavis-mozilla great :+1: do you have any ETA (no mentioned in currents [projects](https://github.com/mozilla/DeepSpeech/projects))?
PS: I would be happy to help cleanup/prepare datasets if needed

**Comment by kdavis-mozilla** [comment]
@nicolaspanel For other languages the timing depends upon the rate at which Common Voice collects data for the language.

For example, a couple of weeks ago data donation started for [French](https://voice.mozilla.org/fr). The faster Common Voice collects data for a particular language, e.g. French, the faster we can bring you a model for that language.

So to a very large extent the timings will be determined be the community of Common Voice.

**Comment by lissyx** [comment]
@nicolaspanel For french we also deeply need to diversify our sources. If you're interested, you can find informations on https://github.com/mozfr/besogne/wiki/Common-Voice-Fr

**Comment by Sorkanius** [comment]
Are there any news on next trained model release?

Thanks a lot, your work is amazing!

**Comment by kdavis-mozilla** [comment]
No real updates yet, but as we just started collecting in other languages we need to give it a bit more time before we have enough data to train on.

**Comment by kdavis-mozilla** [comment]
PS: And thanks for the compliment!

**Comment by Sorkanius** [comment]
Are you working towards increasing the dataset or trying new networks? Or both maybe?

**Comment by kdavis-mozilla** [comment]
Right now trying to get more data through Common Voice

**Comment by kdavis-mozilla** [comment]
Licensing issues prevent this.

**Comment by Sorkanius** [comment]
Oh I understand, thanks again for your time, Keep up the great work!

**Comment by akshat9425** [comment]
Is there any pretrained model of indian english pls send me if it exists

**Comment by b-ak** [comment]
@akshat9425 A couple of us are working towards achieving something on similar lines. We could explore the possibility of collaborating on the same. Write to me bak0@protonmail.com
